.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
-unprocessed/*
\ No newline at end of file
+# ignore files in unprocessed but keep the unprocessed folder for local dev
+unprocessed/*
+!unprocessed/.keep

```

Comment:
Good catch

Reply:
Just need to add the folder and put .keep in it ;). Made the change here: b5ed15c